setup.py

```diff
@@ -26,7 +26,7 @@
     license='BSD',
     packages=['sql_server', 'sql_server.pyodbc'],
     install_requires=[
-        'Django>=2.1.0,<2.2',
+        'Django>=2.2.0,<2.3',
```

Comment:
Just for the record, there will be no Django 2.3. The next release is 3.0 😊
See : https://docs.djangoproject.com/en/dev/internals/release-process/ and https://www.djangoproject.com/download/